viewer/lib/world.js

```diff
@@ -23,7 +23,7 @@ class World {
     this.Chunk = Chunks(version)
     this.columns = {}
     this.blockCache = {}
-    this.biomeCache = mcData(version).biomes
```

Comment:
making a new instance here is no different than using mcdata but with extra copying

Reply:
yeah that's what I thought too

Reply:
but I think Ive seen it done like that elsewhere and I my other pr got blocked for not doing it

Reply:
The registry should be passed here, not the version

Reply:
oh nevermind you mean like this right

Reply:
is it good now ? looks ok to me

Reply:
The World class here is internal, it's not exposed to users and instead instantiated by the n worker threads. So this will make no difference and always create a new Registry copy.

https://github.com/PrismarineJS/prismarine-viewer/blob/6a8f23d2bdfd69e8882864b4ce0483180c263e48/viewer/lib/worker.js#L44

Reply:
ah yeah so then this wouldn't work at all like this

Reply:
we would need to pass the calls to pregistry over the network to propagate updates from packets